core/src/main/java/org/wildfly/extras/creaper/core/online/SslOptions.java

```diff
@@ -174,6 +177,14 @@ public Builder trustStoreType(KeyStoreType trustStoreType) {
             return this;
         }
 
+        /**
+         * Errors during hostname verification will be ignored.
+         */
+        public Builder turnOffHostnameVerification() {
```

Comment:
Hmm, that's funny. I overlooked that. @OndrejKotek, do you have any input on that? I'm not terribly concerned about this, but I'd like Creaper to be usable in all contexts, including people that have stricter security requirements than us.